index.bs

```diff
@@ -308,7 +308,7 @@ spec: HEADER-STRUCTURE; urlPrefix: https://httpwg.org/http-extensions/draft-ietf
   <section>
     <h3 id="default-allowlists">Default Allowlists</h3>
     <p>Every <a>policy-controlled feature</a> has a <dfn export lt=
-    "default allowlist|default allowlists">default allowlist</dfn>. The
+    "default allowlist|default allowlists" for="/">default allowlist</dfn>. The
```

Comment:
Should this be `for="policy-controlled feature"` instead? It's not a top-level concept, but an attribute of `feature`